app/utils/DomainValidator.scala

```diff
@@ -65,7 +65,7 @@ class DomainValidator(registration : Registration) extends ValidationUtil with L
       trustees => {
         val ninoList: List[(String, Int)] = getTrusteesNinoWithIndex(trustees)
         val duplicatesNino = findDuplicates(ninoList).reverse
-        logger.info(s"[indTrusteesDuplicateNino] Number of Duplicate Nino found : ${duplicatesNino.size} ")
+        logger.info(s"[DomainValidator][indTrusteesDuplicateNino] Duplicate Nino's found : ${duplicatesNino.size} ")
```

Comment:
Not super important but plural should not have apostrophe
```suggestion
        logger.info(s"[DomainValidator][indTrusteesDuplicateNino] Duplicate Ninos found : ${duplicatesNino.size} ")
```

Reply:
think this was a copy and paste job I did to standardise the rest of the logs, could have easily made a small change like this